de-lib/src/main/java/org/iplantc/de/client/models/UserInfo.java

```diff
@@ -78,8 +80,41 @@ public void init(String userInfoJson) {
         preferences = userBootstrap.getPreferences();
         session = userBootstrap.getSession();
         appsInfo = userBootstrap.getAppsInfo();
-        workspace = appsInfo.getWorkspace();
-        systemIds = appsInfo.getSystemsIds();
+        if (appsInfo != null) {
+            workspace = appsInfo.getWorkspace();
+            systemIds = appsInfo.getSystemsIds();
+        }
+
+        checkForErrors();
+    }
+
+    private void checkForErrors() {
+        if (userProfile == null || hasUserProfileError()) {
+            logBootStrapError(UserBootstrap.USER_INFO_KEY, null);
```

Comment:
(this is just an OCD thing on my part, feel free to ignore)

One of these says `BootStrap` and one says `Bootstrap`, which maybe isn't likely to confuse but is inconsistent (I'd guess `Bootstrap` is the more common usage here, probably?)